packages/amplication-client/src/ModuleActions/NewModuleAction.tsx

```diff
@@ -87,7 +87,6 @@ const NewModuleAction = ({
     [
       createModuleAction,
       resourceId,
-      moduleId,
```

Comment:
I am not sure why `moduleId` was removed from the dependencies here although it's used

Reply:
moduleId is a parameter of the function and not a dependency